src/App.jsx

```diff
@@ -29,13 +30,16 @@ function App() {
                             path={Routes.ACCOMMODATION_EDIT}
                             component={AccommodationEdit}
                         />
-                    </Switch>
-                    <Switch>
                         <Route
                             path={Routes.GUESTS}
                             exact
                             render={() => <GuestsPage />}
                         />
+                        <Route
+                            path={Routes.HOSTS}
+                            exact
+                            render={() => <HostsPage />}
+                        />
```

Comment:
I've merged this to a single <Switch> otherwise I was not seeing my page component rendered